src/bem-naming-factory/modifiers-mapper-creator.js

```diff
@@ -0,0 +1,27 @@
+import isString from 'lodash/isString';
+import isPlainObject from 'lodash/isPlainObject';
+import isUndefined from 'lodash/isUndefined';
+import isArray from 'lodash/isArray';
+import isEmpty from 'lodash/isEmpty';
+import mapKeys from 'lodash/mapKeys';
+import kebabCase from 'lodash/kebabCase';
+import compact from 'lodash/compact';
+import {assertNamePart} from '../bem-naming-validators';
+import {Config} from '../config';
+
+export function createModifiersMapper(name) {
+    return function mapModifier(modifier) {
+        if (isUndefined(modifier) || (isString(modifier) && isEmpty(modifier)))  {
```

Comment:
And why the check is limited only by undefined and empty string? What about other empty values?